README.md

```diff
@@ -18,10 +23,6 @@
 
 ## Getting Started
```

Comment:
- [OPTIONAL] Kindly, Add a descriptive README file to your project,  A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally as this will be of help to non-technical users.  this includes `git clone URL`, `cd project_name`, etc, please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.